docs/notebooks/delayed-reward.ipynb

```diff
@@ -393,112 +415,266 @@
                         "text/plain": [
-                            "Text(0, 0.5, 'Cumulative Regret')"
+                            "<Figure size 640x480 with 1 Axes>"
                         ]
                     },
-                    "execution_count": 11,
+                    "metadata": {},
+                    "output_type": "display_data"
+                }
+            ],
+            "source": [
+                "plt.plot(np.cumsum(regret))\n",
+                "zero_ninety_eight_decay_regret = np.cumsum(regret)[-1]\n",
+                "plt.xlabel(\"Round\")\n",
+                "plt.ylabel(\"Cumulative Regret\")\n",
+                "plt.title(f\"Cumulative regret for 0.98 decay: {zero_ninety_eight_decay_regret:0.1f}\")\n",
+                "summary_of_regrets['Regret with 0.98 decay rate'] = zero_ninety_eight_decay_regret "
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "Now we'll optimize the `decay_rate` hyperparameter using `optuna`."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": 13,
+            "metadata": {},
+            "outputs": [
+                {
+                    "name": "stderr",
+                    "output_type": "stream",
+                    "text": [
+                        "/Users/cristiandagnino/Library/Caches/pypoetry/virtualenvs/bayesianbandits-B6ZK3PHb-py3.12/lib/python3.12/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
+                        "  from .autonotebook import tqdm as notebook_tqdm\n",
+                        "[I 2024-11-03 13:04:47,762] A new study created in memory with name: no-name-cd809bc2-c512-4f7a-b7eb-d964ca3416f9\n",
```

Comment:
Is it possible to squash stderr here and just show a summary of results or something like that? All of these log lines are quite busy.